src/test/java/de/retest/surili/commons/actionselection/RandomNaughtyStringsInputGeneratorTest.java

```diff
@@ -0,0 +1,30 @@
+package de.retest.surili.commons.actionselection;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.mock;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import de.retest.recheck.ui.descriptors.Element;
+
+class RandomNaughtyStringsInputGeneratorTest {
+	private static final long SEED = 2;
```

Comment:
We could inline this as it isn't needed somewhere else? And could you use a `long` literal (i.e. `2L`)?